src/vconsole/vconsole-setup.c

```diff
@@ -234,38 +225,78 @@ static void setup_remaining_vcs(int fd, int vc_max, bool utf8) {
                 return;
         }
 
+        fontbuf = new(unsigned char, 32*32*512/8);
+        if (!fontbuf) {
+                log_oom();
+                return;
+        }
+
         /* get active, and 16 bit mask of used VT numbers */
         r = ioctl(fd, VT_GETSTATE, &vcs);
         if (r < 0) {
                 log_warning_errno(errno, "VT_GETSTATE failed, ignoring remaining consoles: %m");
                 return;
         }
 
+        /* get fonts from source console */
+        cfo.data = fontbuf;
+        r = ioctl(fd, KDFONTOP, &cfo);
+        if (r < 0) {
+                err_ioctl = "KD_FONT_OP_GET";
+        } else {
+                unimapd.entries  = unipairs;
+                unimapd.entry_ct = USHRT_MAX;
+                r = ioctl(fd, GIO_UNIMAP, &unimapd);
+                if (r < 0)
+                        err_ioctl = "GIO_UNIMAP";
+                else
+                        cfo.op = KD_FONT_OP_SET;
+        }
+        if (r < 0)
+                log_warning_errno(errno, "%s failed, fonts will not be copied: %m", err_ioctl);
```

Comment:
I think it would be simpler and more obvious if you'd get rid of the err_ioctl variable. Simply invoke log_warning_errno() directly where you currently assign err_ioctl. I think it's ultimately simpler and easier to grok that away. And shorter too, as you can drop at least 3 lines of code...